Step2-Temperature.js

```diff
@@ -4,6 +4,11 @@ $(document).ready(function() {
   // showStep(2);
 
   var output = new OutputView({ el: $('.temperature .output') });
+  var temp = 0;
+  getTemperature(32605, function(temp) {
+    // The temp variable is set to some value;
+    // Output the temperature of a given ZIP code to the screen here.
+    output.append("The temperature at 32605 is: " + temp);
+  });
 
```

Comment:
One interesting thing to play with:  if you add ...
```
output.append('After the getTemperature call');
```
... here, what shows up on the screen?